sp800-63c/sec8_security.md

```diff
@@ -5,196 +5,60 @@
-    of the following mechanisms is used:
-
-	1.  The secondary authenticator may contain sufficient entropy that an
-    attacker without direct access to the IdP's random number
-    generator cannot guess the value of a valid secondary authenticator.
-
-	2.  The secondary authenticator may contain timely assertion data that
-    is signed by the IdP or integrity protected using a key shared
-    between the IdP and the RP.
-
-
-
--   *Secondary authenticator capture* - To mitigate this threat,
-    adequate protections are in place throughout the lifetime of
-    any secondary authenticators used in the assertion protocol.
-
-	1.  In order to protect the secondary authenticator while it is in
-    transit between the IdP and the subscriber, the secondary
-    authenticator is sent via an authenticated protected channel established
-    during the primary authentication of the subscriber. 
-
-	2.  In order to protect the secondary authenticator from capture as it
-    is submitted to the RP, the secondary authenticator is used in
-    an authentication protocol which protects against eavesdropping and
-    man-in-the-middle attacks.
-
-	3.  In order to protect the secondary authenticator after it has been
-    used, it is never transmitted over an unprotected channel or to
-    an unauthenticated party while it is still valid.
-    
--   *Assertion substitution* - To mitigate this threat, one of the
-    following mechanisms is used:
-
-	1.  Responses to assertion requests contain the value of the assertion reference used
-    in the request or some other nonce that was cryptographically bound
-    to the request by the RP.
-
-	2.  Responses to assertion requests are bound to the corresponding
-    requests by message order, as in HTTP, provided that assertions and
-    requests are protected by a protocol such as TLS that can detect and
-    disallow malicious reordering of packets.
+Because of the additional party, the IdP, in the authentication process, there are additional opportunities for attackers to compromise federated identity transactions. This section summarizes many of the attacks and their mitigations that are applicable when federation is being used.
+
+###8.1. Federation Threats
+
+As in non-federated authentication, the motivation of attackers is typically to gain access (or a greater level of access) to a resource or service provided by an RP. Attackers may also attempt to impersonate a subscriber. Rogue or compromised IdPs, RPs, user endpoints, and parties outside of a typical assertion transaction are potential attackers. To accomplish their attack, they might intercept or modify assertions and assertion references. Furthermore, two or more entities may attempt to subvert assertion protocols by directly compromising the integrity or confidentiality of the assertion data. For the purpose of these types of threats, authorized parties who attempt to exceed their privileges are considered attackers.
```

Comment:
We tend to use "federation protocol" and "federation transaction" instead of "assertion transaction" and "assertion protocol". This also appears below.